src/file.ts

```diff
@@ -73,7 +73,12 @@ import {HashStreamValidator} from './hash-stream-validator';
 import {URL} from 'url';
 
 import retry = require('async-retry');
-import {SetMetadataOptions} from './nodejs-common/service-object';
+import {
+  DeleteCallback,
+  DeleteOptions,
+  SetMetadataOptions,
+} from './nodejs-common/service-object';
+import * as r from 'teeny-request';
```

Comment:
what is this import being used for?

Reply:
It uses the type in the return https://github.com/googleapis/nodejs-storage/pull/2014/files#diff-bc9705d0f7a567399044dfc66ccc82d4d9aa1cff116842a0094d54e463c9ecbcR2056